tests/infer/reparam/test_stable.py

```diff
@@ -55,7 +58,7 @@ def model():
         expected_grads = grad(expected_m.sum(), params, retain_graph=True)
         actual_grads = grad(actual_m.sum(), params, retain_graph=True)
         assert_close(actual_grads[0], expected_grads[0], atol=0.2)
-        assert_close(actual_grads[1], expected_grads[1], atol=0.1)
+        assert_close(actual_grads[1][skew != 0], expected_grads[1][skew != 0], atol=0.1)
```

Comment:
this test is unstable when skew ~ 0 (say skew < 1e-3). Here we only want to make sure that nan does not happen.